enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -699,20 +712,14 @@ still be created to add a namespace to a mesh.)
 
 #### Can we use the Gateway API v1beta1 CRDs?
 
-OSSM 2.3 is based on Istio 1.14, which only supports Gateway API v1alpha2
-whereas OSSM 2.4 is based on Istio 1.16, which supports Gateway API v1beta1.
-Supporting v1beta1 is highly desirable, and OSSM 2.4 brings many other changes
-that are of interest for this enhancement.  If we continue to use OSSM 2.3, we
-will need to continue to install the Gateway API v1alpha2 CRDs.  We need to
-determine whether we will be able to use OSSM 2.4 for this enhancement.
-
-If OSSM 2.4 is not released in time for OpenShift 4.13, we may resort to using a
-pre-release development build of OSSM 2.4 for dev preview.  While not ideal,
-OSSM 2.4 has sufficient advantages to outweigh the disadvantages of using a
-development build in the context of a dev preview.
+_Note: This question was proposed before OSSM 2.3 (Istio 1.14) supported
```

Comment:
The fact that Istio 1.14 is already end-of-life should be mentioned here.  it is one of the main cons of using OSSM 2.3 now.  I think you should update the original question with this information and other new facts, not remove it.

Reply:
> The fact that Istio 1.14 is already end-of-life should be mentioned here.

Okay I can mention that.

> I think you should update the original question with this information and other new facts, not remove it.

I can put it back, but I was struggling because the question definition didn't make a lot of sense now that it has incorrect information, but let me try to wordsmith it up.